stdlib/private/SwiftPrivatePthreadExtras/SwiftPrivatePthreadExtras.swift

```diff
@@ -125,7 +125,7 @@ public class _stdlib_Barrier {
   }
 
   deinit {
-    _stdlib_pthread_barrier_destroy(_pthreadBarrierPtr)
+    _ = _stdlib_pthread_barrier_destroy(_pthreadBarrierPtr)
```

Comment:
Thanks! This is exactly the kind of feedback I was hoping for. I wasn't sure what returns were intentionally ignored and which were accidental.